### DISCUSSION TITLE
这个平台啥时候建立的？主要是啥内容啊（）

### DISCUSSION THREAD

**Discussion by Dingdong1018** [share121/inter-knot#5160] [其他]
刚在哔哩哔哩刷到了app，然后就下载了

**Comment by Dave-12138** [comment]
只要是不让大家反感的话题，什么都可以聊<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Comment by TNCNET** [comment]
类似于绝区零中的绳网（一个游戏中的虚拟社交平台），这里算是复刻的。
tips：
1.不能发布敏感/NSFW这些内容。
2.帖子不能引战/攻击/泄露他人个人信息（俗称开盒）等行为。
这些是在任何公开平台上都不行的。并且本身就是不应该做的。

**Comment by TNCNET** [comment]
至于你想聊的，除去我提供的建议以外的，正常的内容都可以，包括讨论游戏配队，扩列什么的。
